artemis-distribution/src/main/resources/licenses/bin/licenses/LICENSE-angular-resizable.txt

```diff
@@ -0,0 +1,7 @@
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.  No representations are made about the suitability of this
+software for any purpose.  It is provided "as is" without express or
+implied warranty.
```

Comment:
This seems a bit broken as it references retaining a copyright notice that doesnt exist..was there an alternative/complete text available?

Reply:
@gemmellr the angular-resizable project owner doesn't want to provide a copyright notice see https://github.com/Reklino/angular-resizable/issues/82

Reply:
Mmm, guess theres not much we can do about that then. Seems an odd licence text to use given that though, and doesnt change who wrote what at all. I guess we did retain the non-existant notice space as intended though hehe.

Reply:
I could replace `For details, see licenses/LICENSE-angular-resizable.txt` with `No license file provided`

Reply:
@gemmellr I have just pushed a commit to remove the `licenses/LICENSE-angular-resizable.txt`file and to add the comment `No text license provided by the author`

Reply:
No, we definitely shouldnt do that, as it does actually have a licence, an MIT variant, and it needs to or we probably cant use it. Just put it back the way it was, as silly as that is.

Reply:
I created the `licenses/LICENSE-angular-resizable.txt` by myself copying it from https://fedoraproject.org/wiki/Licensing:MIT because the product is available under the MIT license (see https://github.com/Reklino/angular-resizable#license) but I can not find any license file at https://github.com/Reklino/angular-resizable
The MIT license refers to the copyright notice so there is no generic valid MIT license that we can use
I would write:
`
==============================================================================
For angular-resizable:
==============================================================================
This product bundles angular-resizable, which is available under a "MIT" license.
No text license provided by the author
`

Reply:
I created the `licenses/LICENSE-angular-resizable.txt` by myself copying it from https://fedoraproject.org/wiki/Licensing:MIT because the product is available under the MIT license (see https://github.com/Reklino/angular-resizable#license) but I can not find any license file at https://github.com/Reklino/angular-resizable
The MIT license refers to the copyright notice so there is no generic valid MIT license that we can use
I would write:
`
==============================================================================
For angular-resizable:
==============================================================================
This product bundles angular-resizable, which is available under a "MIT" license.
No text license provided by the author
`

Reply:
I created the `licenses/LICENSE-angular-resizable.txt` by myself copying it from https://fedoraproject.org/wiki/Licensing:MIT because the product is available under the MIT license (see https://github.com/Reklino/angular-resizable#license) but I can not find any license file at https://github.com/Reklino/angular-resizable
The MIT license refers to the copyright notice so there is no generic valid MIT license that we can use
I would write:
`
==============================================================================
For angular-resizable:
==============================================================================
This product bundles angular-resizable, which is available under a "MIT" license.
No text license provided by the author
`

Reply:
I think thats a little dodgy really.

Reply:
Finally I have found an example using `angular-resizable` created by the same author with the same code and with a valid MIT license at https://codepen.io/Reklino/pen/raRaXq/

Reply:
Great. Somewhat bizarre, but great.